hw_zolotikov/calculator.py

```diff
@@ -0,0 +1,27 @@
+def mymult(x, y):
+        return x * y
+def mysub(x, y):
+        return x - y
+def mysum(a, b):
+    return a+b
+def mydiv(x,y):
+    return x/y
+def main():
```

Comment:
Между определениями функций принято оставлять 2 пустые строки ([ссылка](https://peps.python.org/pep-0008/#blank-lines)). Тут это не критично, но с ДЗ 4 может влиять на оценку. 
```suggestion
    return x/y
    
    
def main():
```